### DISCUSSION TITLE
[NOT-BUG] ¿Cómo puedo ver el error completo de un bot de código custom?

### DISCUSSION THREAD

**Discussion by seiturra** [Cotalker/documentation#248] [2. Questions and Answers] [answered]
Buenas,

Un bot de un cliente tiene una etapa custom que se comunica con SAP y ha estado ejecutándose hace mucho tiempo. Hace unos días está fallando en la etapa de integración pero el mensaje de Automation Log no es de mucha ayuda(status: ERROR, errors: [Runtime Check Data Failed]). Quisiera saber si hay alguna manera de ver el error con más detalle sin tener que levantar un ambiente local con la integración para replicar la falla y ver en mi consola dónde falla.

Saludos.

**Comment by rafaelwalter** [comment]
@guillevalin

**Comment by guillevalin** [comment, marked as answer]
@SeIturra el bot personalizado desarrollado por ustedes realiza una validación sobre un campo que en la programación de su bot paramétrico lo envían en blanco, es por ello el "error".

Antes del refactor de los bots paramétricos no revisabamos checkData() y por eso funcionaba.
